src/partitioner/__main__.py

```diff
@@ -1,9 +1,24 @@
 """Main method to enable command line execution"""
+
 import sys
 
+import dask_runner
+import single_runner
 from arguments import PartitionArguments
 
+
+def run():
+    """Pick a runtime environment and run the partitioner"""
+    match args.runtime:
+        case "single":

```

Comment:
should we/have you set "single" as the default case, and following that do we/should we have a set of default arguments for the partitioner?